scripts/helpers.sh

```diff
@@ -36,14 +36,44 @@ function setGitVars() {
   changelog=$(git log --pretty='%b' "${last_release_tag}"..HEAD --grep="${package}" ${greps} --reverse | grep -E '^Changelog:' | awk '{gsub("Changelog:", "-", $0); print}')
 }
 
+
+function gitPush() {
+  git checkout -b $1
+  git push
+  git push --tags
+  git checkout ${current_branch}
+  echo "${color_green}Commits and tags pushed!${color_reset}"
+}
+
 function checkPackage() {
   if [[ ! "${package}" ]] || [[ ! -d "packages/${package}" ]]; then
      echo "${color_red}Please provide a valid package name${color_reset}"
      exit
   fi
 }
 
+function setNiceVersion() {
+  nice_version=$(head -n 1 nice-current-version.txt | (read s;echo ${s//[(.0).]/}))
+}
+
 function setCurrentReleaseTag() {
-  nice_version=$(head -n 1 nice-current-version.txt)
-  release_tag="nice${nice_version//[(.0).]/}"
+  setNiceVersion
+  release_tag="nice${nice_version}"
+}
+
+function setNextVersion() {
+  if [[ $(git rev-parse --abbrev-ref --symbolic-full-name @{u} | cut -d '/' -f2-) == "master" ]]; then
```

Comment:
Maybe throw an error if branch is not master or nice-releases/* because then someones tries to publish a package on a feature branch?